javascript/javascript1/week2/flight-booking.js

```diff
@@ -0,0 +1,24 @@
+let firstname,surname,FormalName,gender;
+function getFullname(firstname,surname,gender,FormalName)
+{
+    if (gender==="male" && FormalName==="true")
```

Comment:
FormalName should be `boolean` not string.
so we can check its value like this `FormalName === true` 
boolean value can be `true` or `false` not `"true"` or `"false"` as strings